AssetRipper.Fundamentals/Utils/UnityPatchUtils.cs

```diff
@@ -0,0 +1,32 @@
+using System.IO;
+using System.Reflection;
+using System.Text;
+
+namespace AssetRipper.Core.Utils
+{
+	public static class UnityPatchUtils
```

Comment:
Move this class to the `AssetRipper.Core` project. Only the bare minimum goes into `AssetRipper.Fundamentals`.